include/bluetooth/direction.h

```diff
@@ -9,20 +9,35 @@
 
 /** Constant Tone Extension (CTE) types. */
 enum bt_df_cte_type {
+	/** Angle of Arrival mode. Antenna switching done on receiver site. */
+	BT_DF_CTE_TYPE_AOA = 0,
+	/** Angle of Departure mode with 1 us antenna switching slots.
+	 *  Antenna switching done on transmitter site.
+	 */
+	BT_DF_CTE_TYPE_AOD_1US = 1,
```

Comment:
Consider the following to avoid having long `brief`s:

```suggestion
	/** 
	 * @brief Angle of Departure mode with 1 us antenna switching slots.
	 *
	 * Antenna switching done on transmitter site.
	 */
	BT_DF_CTE_TYPE_AOD_1US = 1,
```

(same comment applies below as well)

Reply:
Corrected